jj/logs/formatters/_simple_formatter.py

```diff
@@ -1,21 +1,37 @@
+import os
 from logging import LogRecord
-from os import linesep
 
 from ...requests import Request
 from ...responses import Response
 from ._formatter import Formatter
 
 __all__ = ("SimpleFormatter",)
 
+INFO = 'info'
+DEBUG = 'debug'
+TRACE = 'trace'
+
+log_level = os.getenv('MOCK_LOG_LEVEL', INFO).lower()
+body_output_limit = int(os.getenv('MOCK_BODY_OUTPUT_LIMIT', 10 ** 6))
+
 
 class SimpleFormatter(Formatter):
-    def format_request(self, request: Request, record: LogRecord) -> str:
-        return "-> {} {}".format(request.method, request.url.path)
-
-    def format_response(self, response: Response, request: Request, record: LogRecord) -> str:
```

Comment:
In the case of simultaneous request/response queries, the logs get mixed up, so the complete log record is written to `format_response`. `format_request` is retained for backward compatibility.